src/views/v2/month/calendar-body/day/calendar-events/calendar-event/featured-image.php

```diff
@@ -29,14 +29,18 @@
 		rel="bookmark"
 		class="tribe-events-calendar-month__calendar-event-featured-image-link"
 	>
-		<div class="tribe-events-calendar-month__calendar-event-featured-image tribe-common-c-image tribe-common-c-image--bg">
-			<div
-				class="tribe-common-c-image__bg"
-				style="background-image: url('<?php echo esc_url( $event->thumbnail->full->url ); ?>');"
-				role="img"
-				aria-label="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
-			>
-			</div>
-		</div>
+		<img
+			src="<?php echo esc_attr( $event->thumbnail->full->url ); ?>"
```

Comment:
:no_entry_sign: Wrong escaping function. href and src attributes should be escaped by `esc_url()`, not by `esc_attr()`.

<sub>hash: a062848c00565629cae84c0ee40e3fcf</sub>